### DISCUSSION TITLE
CRR MUI

### DISCUSSION THREAD

**Discussion by sahithya2402** [ercot/api-specs#32] [Q&A]
Hi, 
I am looking for resources that can assist me in navigating the ERCOT CRR MUI via API. Specifically, I am trying to access and download the private results, in addition to automating the process of submitting CRR bids.

**Comment by ajay-ercot** [comment]
Hello, we currently don't have any published API's for CRR today and the external interaction with the application is only via the user interface. But this is something we will consider for our future product roadmap.
